misk-jaeger/src/main/kotlin/misk/tracing/backends/jaeger/MDCScopeManager.kt

```diff
@@ -0,0 +1,63 @@
+package misk.tracing.backends.jaeger
+
+import io.opentracing.Scope
+import io.opentracing.ScopeManager
+import io.opentracing.Span
+import org.slf4j.MDC
+
+/**
+ * [MDCScopeManager] is a [ScopeManager] that records the current trace-id, span-id, parent-id
+ * in the current logging [MDC]. It is jaeger specific because getting access to these values
+ * is dependent on the specific tracing vendor implementation.
```

Comment:
in the future could probably just have some interface that provides the MDC fields, since the majority of this manager has nothing to do with jaegar

Reply:
agreed

Reply:
well, sortof - when trace-ids/span-ids/parent-ids are available is actually highly dependent on the implementation. some implementations don't assign them until a span is finished and sent to the reporter

Reply:
lol standards